components/eamxx/tests/single-process/ace/CMakeLists.txt

```diff
@@ -0,0 +1,63 @@
+# Initialize PyTorch found status to FALSE
+set(SCREAM_FOUND_PYTORCH FALSE CACHE BOOL "Whether PyTorch was found" FORCE)
+
+# Use a configurable path for PyTorch with a default value
+if(NOT DEFINED TORCH_CMAKE_PATH)
```

Comment:
Again, this cmake code should not be needed. Linking against the `ace` physics lib should transitively link againt torch already.